Client/Frontend/Browser/TabTrayButtonExtensions.swift

```diff
@@ -29,7 +29,9 @@ class PrivateModeButton: ToggleButton, NotificationThemeable, PrivateModeUI {
         accessibilityValue = isSelected ? .TabTrayToggleAccessibilityValueOn : .TabTrayToggleAccessibilityValueOff
     }
 
-    func applyTheme() {
+    func applyTheme(theme: Theme) {
```

Comment:
I had started in a branch to add a `ThemeApplicable` protocol, so views that are themeable follows a protocol so we can easily update them through a standard interface. Since the rest of my work isn't ready, I could maybe just open a PR for that ?

Reply:
I totally forgot we were planning that. I'll merge this PR and then go back over the tab tray to use the new protocol whenever it's ready.